EncryptionTests/Encryption Tests/CDTQIndexManagerEncryptionTests.m

```diff
@@ -209,4 +210,21 @@ - (void)testCreateQueryIndexManagerWithFixedKeyProviderFailsIfDBIsEncryptedButKe
     XCTAssertNotNil(err, @"There is an error");
 }
 
+- (void)testTextSearchIsAvailable
+{
+    CDTHelperFixedKeyProvider *provider = [CDTHelperFixedKeyProvider provider];
+    CDTDatastore *datastore = [self.factory datastoreNamed:@"is_fts_activated"
+                                 withEncryptionKeyProvider:provider
+                                                     error:nil];
+    
+    CDTQIndexManager *im = [[CDTQIndexManager alloc] initUsingDatastore:datastore error:nil];
+
+    NSSet *compileOptions = [DBQueryUtils compileOptions:im.database];
+    NSSet *ftsCompileOptions = [NSSet setWithArray:@[ @"ENABLE_FTS3", @"ENABLE_FTS3_PARENTHESIS" ]];
```

Comment:
@alfinkel: You are right, I copied this test from [here](https://github.com/cloudant/CDTDatastore/blob/master/Tests/Tests/CDTQIndexManagerTests.m#L249)